src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const oneDay = 40;
+  const discountSevenDays = 50;
+  const discountThreeDays = 20;
+  let total = oneDay * days;
+
+  if (days >= 7) {
+    total -= discountSevenDays;
+  } else if (days >= 3) {
+    total -= discountThreeDays;
+  } else {
+    total = oneDay * days;
```

Comment:
This line is unnecessary. You have already calculated `total` at line 9, so there's no need to do it again. You can remove line 16. 😊